examples/Spec/IntCompact.hs

```diff
@@ -29,13 +31,14 @@ instance Description IntDescr Int where
         | i < 11 && i > -111 = Small
         | otherwise = Large {isBound = i == minBound || i == maxBound}
 
-  descriptionGen Zero = pure 0
-  descriptionGen (Positive (Large True)) = pure maxBound
-  descriptionGen (Positive (Large False)) = int (linear 11 (maxBound - 1))
-  descriptionGen (Positive Small) = int (linear 1 10)
-  descriptionGen (Negative (Large True)) = pure minBound
-  descriptionGen (Negative (Large False)) = int (linear (minBound + 1) (-11))
-  descriptionGen (Negative Small) = int (linear (-10) (-1))
+  genForDescription = \case
```

Comment:
We should align on '->', vim's EasyAlign does that automatically, vscode probably has something similar

Reply:
Every time I run `make format` it changes it back. I don't think it's worth maintaining against the formatter..